core/trino-parser/src/test/java/io/trino/sql/parser/TestSqlParser.java

```diff
@@ -360,12 +360,22 @@ public void testGenericLiteral()
         assertGenericLiteral("foo");
     }
 
+    private static void assertGenericLiteral(String type)
```

Comment:
Unrelated move

Reply:
move is important, but yes, can make it a separate commit if you want